espaloma/data/qcarchive_utils.py

```diff
@@ -185,7 +185,8 @@ def get_graphs(collection, record_names, spec_name="default"):
             collection.iterate_records(record_names, specification_names=[spec_name]),
             collection.iterate_entries(record_names),
     ):
-        g = process_record(record, entry)
+        # note interate records returns a tuple of lenth 3 (name, spec_name, actual record)
+        g = process_record(record[2], entry)
```

Comment:
Since this is a one-line change, does it make sense to use `try...except` to make this compatible with old and new style QCPortal? Or does the old-style no longer exist?

Reply:
This change here was just a copy-paste error on my part.  No need for a `try..except` here.  I don't think we want to build in compatibility with the old qcportal syntax, since at some point, the molssi servers will stop hosting the old database/software (so there will be nothing to connect to).

Reply:
+1 to only supporting newest qcportal